### ISSUE TITLE
Move CI to cibuildwheel

### ISSUE THREAD

**Issue by mattip** [vmprof/vmprof-python#241]
Related to #223, #240. We have not released in quite a while. Since then travis-ci.org has become travis-ci.com and makes it expensive (in terms of credits) to build on macOS. Many projects have moved to [cibuildwheel](https://cibuildwheel.readthedocs.io/en/stable/) on top of github actions to build for x86_64, leaving only arm64/ppc/s390x on travis (I am pretty sure these are actually in [the "Partner Queue" credit category](https://docs.travis-ci.com/user/billing-overview/#usage---credits) and are free)

Bottom line: we should move windows, macOS and x86_64 linux/musl to cibuildwheel. This would also make it easier to trigger a release wheel build.

**Comment by mattip** [comment]
Still todo:
- make builds pass on macOS, windows 
- cibuildwheel can upload directly to PyPI. Should it? Alternatively: the artifact (from the [bottom of the [build page](https://github.com/vmprof/vmprof-python/actions/runs/1413309260)) can be downloaded and manually uploaded to PyPI.

**Comment by oberstet** [comment]
cibuildwheel sounds neat! wasn't aware of that latest thingy. fwiw, rgd "upload to pypi", I'm using https://pypi.org/project/twine/ which works nice and robust

**Comment by mattip** [comment]
Closing, cibuildwheel wheels are now on PyPI
